sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_train_geneformer.py

```diff
@@ -37,7 +37,8 @@ def test_bionemo2_rootdir():
     assert data_path.is_dir(), "Test data directory is supposed to be a directory."
 
 
-def test_main_runs(tmpdir):
```

Comment:
can we make sure this is running a super small version of the model? Any other CLI options we can change to make this run as fast as possible would be great.